Source/iOS/App/DolphiniOS/UI/Emulation/EmulationiOSViewController.mm

```diff
@@ -197,6 +243,10 @@ - (bool)isWiimoteTouchPadAttached {
   return true;
 }
 
+- (bool)emulateSkylanderPortal {
```

Comment:
This method doesn't seem to be used anywhere. Is this supposed to be used for conditionally adding the "Skylanders Portal" `UIAction` above?

Reply:
Ah yes - think I had this in to try to conditionally display the menu item, was going to ask if you would be happy to convert the UIMenu items to instead be added with a UIMenuBuilder